pkg/topology/builder.go

```diff
@@ -106,13 +106,22 @@ func (b *Builder) evaluateTrafficTarget(res *resources, topology *Topology, tt *
 			Service:       svcKey,
 			TrafficTarget: Key{tt.Name, tt.Namespace},
 		}
-		topology.ServiceTrafficTargets[svcTTKey] = stt
 
 		svc, ok := topology.Services[svcKey]
+		// In the current version of the spec, the traffic will always go to a TrafficSplit. So we don't need to evaluate
+		// Pods if there's already a TrafficSplit on the Service.
```

Comment:
```suggestion
		// Pods if there is already a TrafficSplit on the Service.
```

Also I'm not sure I understand this sentence. It's missing a verb?

Reply:
This is not the begining of the sentence. This is why there was no `.` at the end of the previous line ;)